doc/qa_docs/str.rst

```diff
@@ -0,0 +1,246 @@
+state, the processes will be documented in the test  traceability matrix (TTM) in
+:numref:`Section %s <ref_armi_test_trace_matrix>`  (The TTM provides traceability for each test to
+the required criteria). Otherwise, there are no special inputs necessary to run test cases or steps
+to  restore the system. 
+
+Required Ranges of Input Parameters for the Test Case(s)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+If a test uses a range of inputs, then it will be documented in the TTM in 
+:numref:`Section %s <ref_armi_test_trace_matrix>`. Otherwise, there are no required ranges of inputs
+for the test case.
+
+Expected Results for the Test Case(s)
+"""""""""""""""""""""""""""""""""""""
+
+If a test expects a specific result, it will be documented in the TTM in 
+:numref:`Section %s <ref_armi_test_trace_matrix>`. Otherwise, the expected test result is that no
+error is raised, which constitutes a passing test.
+
+Acceptance Criteria for the Test Case(s)
+""""""""""""""""""""""""""""""""""""""""
+
+The acceptance criteria for the test cases will be described. In cases where the SRSD requirement
+acceptance criteria is acceptable for the test case  acceptance criteria, the SRSD requirement
+acceptance criteria can be referenced  by default.
+
+.. _ref_armi_record_criteria:
+
+Test Record Criteria
+^^^^^^^^^^^^^^^^^^^^
+
+The default values for the remaining 12 criteria pertaining to the test record are given in this
+section below. A test record will be produced after the test  is run which contains pertinent
+information about the execution of the test.  This test record will be saved as part of the software
+test report (STR).
+
+Software Tested, Including System Software Used and All Versions
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The ARMI version will be shown in the test record via standard output logs. 
+
+Compute Platform and Hardware Used
+""""""""""""""""""""""""""""""""""
+
+The test record will reference the environment upon which the test is run. See
+:numref:`Section %s <ref_armi_hardware_integration>` for acceptable test environments.
+
+Test Equipment and Calibrations
+"""""""""""""""""""""""""""""""
+
+Not applicable for the ``ARMI`` software.
+
+.. _ref_armi_run_env:
+
+Runtime Environment Including System Software, and Language-Specific Environments
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The runtime environment including the operating system, hardware, and software configuration will be
+specified in the test report. If necessary, more detail will be provided for individual tests which
+utilize custom runtime environments or have dependencies such as custom compiler options.
+
+Date of Test
+""""""""""""
+
+The date of the test execution is recorded in the output of the test.
+
+Tester or Data Recorder
+"""""""""""""""""""""""
+
+Automation will be created to run the acceptance tests.
```

Comment:
nitpick
Maybe something more like "Acceptance tests will be run via automation"?

Reply:
done